specifications/api/swagger.yaml

```diff
@@ -7379,6 +7379,24 @@ paths:
             schema: *ApiSharedWalletPatchData
       responses: *responsesPatchSharedWallet
 
+  /shared-wallets/{walletId}/transactions-construct:
+    post:
+      operationId: constructSharedTransaction
+      tags: ["Transactions New"]
```

Comment:
This should go rather into 'Transactions Shared' under 'Shelley (Shared Wallets)'.